docs/UserGuide.adoc

```diff
@@ -274,7 +274,8 @@ Format: `undo`
 
 [NOTE]
 ====
-Undoable commands: those commands that modify the address book's content (`add`, `delete`, `edit` and `clear`).
+Undoable commands: those commands that modify the address book's content (`add`, `delete`, `edit`, `removeTag +
+` and `clear`).
```

Comment:
the last part should be "`removeTag` and `clear`" (i.e. the ` should follow the removeTag directly, and not be on a new line). This will affect the display in the User Guide (feel free to click `View` above to verify).

Reply:
sorry brainfarted